src/App.jsx

```diff
@@ -2,13 +2,18 @@ import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import HomePage from './components/Home/HomeComponent';
 import LoginComponent from './components/Auth/LoginComponent';
+import SignUpContainer from './containers/Auth/SignUpContainer';
 import './App.scss';
 
+
 const App = () => (
-  <Switch>
-    <Route path="/" exact component={HomePage} />
-    <Route path="/login" exact component={LoginComponent} />
-  </Switch>
+  <div>
+    <HomePage />
+    <Switch>
+      <Route path="/login" exact component={LoginComponent} />
+      <Route path="/signup" component={SignUpContainer} />
```

Comment:
Add the `exact` prop to this route as well

Reply:
Done